components/badge/badge_pins.h

```diff
@@ -140,6 +140,40 @@
 #define MPR121_PIN_NUM_LEDS     10
 #define MPR121_PIN_NUM_SD_CD    11
 
+#elif defined(CONFIG_SHA_BADGE_V3_LITE)
```

Comment:
Too bad about the duplication but adding logic would make it harder to follow, so :+1: